app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
That's a great idea!

We can make the right click context menu specific to the Hex & the Value.

Right click on value to get all formats mentioned in config file. Right click on hex to get copy color in different color formats. Right click anywhere else doesn't open the context menu.

Left click on value to copy the first format in the config list.

Sounds good?